src/dotnet/CarbonAware.DataSources.WattTime/RegionConverter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CarbonAware.Tools.WattTimeClient.Model;
+
+namespace CarbonAware.DataSources.WattTime;
+
+/// <summary>
+/// Class RegionConverter.
```

Comment:
Why not inherit here from the interface?

Reply:
Fixed.